lib/mozart_fetcher/local_cache.ex

```diff
@@ -1,8 +1,22 @@
 defmodule MozartFetcher.LocalCache do
-  def get_or_store(id, f) do
-    case MozartFetcher.environment() do
```

Comment:
Does this mean we now store stuff in the cache when running the unit tests? The services do not restart between tests automatically, could the cache cause issues?

Reply:
discussed face 2 face, but shortly, given that we use caching in a single layer coudl make it seems to make more sense to cache in all environment, al least for now.

Reply:
we'll regret and fix thsi when issues with testing downstream services will arise